src/vsg/viewer/Viewer.cpp

```diff
@@ -212,6 +212,7 @@ class CollectSecondaryCommandGraph : public ConstVisitor
 {
 public:
     vsg::CommandGraphs _secondaries;
+    std::vector<std::shared_ptr<std::mutex> > _execcommandmuters;
```

Comment:
A quick general comment, names need to be human readable ratherthanallbuncheduptogethermakingithardertoreadonaglance.  The standard way to make it readable isToUseCapitalization or_underscore_separated.

Reply:
indeed...fixed